tests/yaml/linter.py

```diff
@@ -0,0 +1,89 @@
+import sys
+from pprint import pprint
+
+import yaml
+
+sys.path.insert(0, '../../src')
+sys.path.insert(0, '../..')
+
+from tests.functional.test_yaml import convert_to_dict
```

Comment:
You probably want to add a `# isort: skip` comment on this line.

Reply:
Good point!